prisma/seeds/prices.ts

```diff
@@ -1,35 +1,127 @@
-import { Prisma } from '@prisma/client'
-export const prices = [
-  { // XECUSD
-    value: new Prisma.Decimal('0.00004095'),
-    createdAt: new Date(),
-    updatedAt: new Date(),
-    timestamp: 1664593200,
-    networkId: 1,
-    quoteId: 1
-  },
-  { // XECCAD
-    value: new Prisma.Decimal('0.00005663'),
-    createdAt: new Date(),
-    updatedAt: new Date(),
-    timestamp: 1664593200,
-    networkId: 1,
-    quoteId: 2
-  },
-  { // BCHUSD
-    value: new Prisma.Decimal('117.9081'),
-    createdAt: new Date(),
-    updatedAt: new Date(),
-    timestamp: 1664593200,
-    networkId: 2,
-    quoteId: 1
-  },
-  { // BCHCAD
-    value: new Prisma.Decimal('163.0735'),
-    createdAt: new Date(),
-    updatedAt: new Date(),
-    timestamp: 1664593200,
-    networkId: 2,
-    quoteId: 2
+import { Price, Prisma } from '@prisma/client'
+import { getAllPricesByTicker } from 'services/priceService'
+import { PRICE_FILE_MAX_RETRIES, TICKERS, NETWORKS, QUOTES, KeyValueAny } from 'constants/index'
+import moment from 'moment'
+
+import * as fs from 'fs'
+import * as path from 'path'
+import { promisify } from 'util'
+
+interface PriceFileData {
+  ticker: string
+  date: string
+  priceInCAD: string
+  priceInUSD: string
+}
+
+export const PATH_PRICE_CSV_FILE = path.join('prisma', 'seeds', 'prices.csv')
+
+async function writeToFile (fileName: string, content: PriceFileData[]): Promise<void> {
+  const writeFile = promisify(fs.writeFile)
+
+  const headers = ['ticker', 'date', 'priceInCAD', 'priceInUSD']
```

Comment:
Agreed, that would be a nice solution, but we have conflicts in package versions when trying to install this package: 
![photo_5152556271285677094_y](https://user-images.githubusercontent.com/124014001/219718450-1927fc22-172b-4ab6-ab4b-f1692ab8e2e2.jpg)

It seems it could be quite a lot of work or version break something to avoid repeating the interface definition here

Reply:
Alright, best that we update the node in another issue and maybe refactor it then